### PR TITLE
RHPAM-2085 : Dropdowns in Security perspective can not be controlled by…

### PR DESCRIPTION
**Pull request by RishiRajAnand**
… keyboard

- Added the feature to select the options using UP and DOWN arrow keys.
- The cause of the issue was using the bootstrap-select class on the container which also add an additional keydown event provided by the bootstrap-select library.
- bootstrap-select class was replaced with a custom class persisting the same look and feel.

### CODE REVIEW

**Comment by kiereleaseuser3** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by pefernan** [comment]
ok to test

**Comment by pefernan** [comment]
jenkins execute full downstream build please

**Comment by pefernan** [comment]
@jomarko would you mind review this?

**Comment by pefernan** [comment]
@ederign could you please take a look?

File: uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/dropdown/items/LiveSearchSelectorDropDownItemViewImpl.java [hunk 1]
```diff
@@ -97,12 +100,23 @@ public void setMultiSelect(boolean multiSelect) {
...
     }
+
+    @EventHandler("itemAnchor")
+    public void onItemClick(ClickEvent event) {
+        onItemSelected(event);
+    }
+
+    @EventHandler("itemAnchor")
+    void onEnterKeyDown(KeyDownEvent event) {
+        if (event.getNativeKeyCode() == 13) {
```
Comment (jomarko) [inline uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/dropdown/items/LiveSearchSelectorDropDownItemViewImpl.java]: Could we please use https://github.com/gwtproject/gwt/blob/master/user/src/com/google/gwt/event/dom/client/KeyCodes.java#L271 ?

Comment (RishiRajAnand) [reply uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/dropdown/items/LiveSearchSelectorDropDownItemViewImpl.java]: Thanks @jomarko for the observation. I'll add the change.

**Comment by domhanak** [comment]
jenkins execute full downstream build

**Comment by domhanak** [comment]
Looks good to me. I tried the steps to reproduce with the changes and no error was shown. I was able to control the dropdowns with UP and DOWN keys.

@RishiRajAnand 
I am not an expert on CSS so, if you know some1 who can review that part, feel free to tag him/her in.

File: uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/dropdown/LiveSearchDropDownView.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
-<div data-field="mainPanel" class="btn-group bootstrap-select">
-    <button data-field="dropDownButton" type="button" class="btn dropdown-toggle btn-default" data-toggle="dropdown" aria-expanded="false">
-        <span id="dropDownText" class="filter-option pull-left appformer-live-search-button-text"></span>&nbsp;<span class="bs-caret"><span class="caret"></span></span>
+<div data-field="mainPanel" class="dropdown appformer-live-search-spinner-select btn-group">
+    <button data-field="dropDownButton" class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown" aria-expanded="false">
+            <span id="dropDownText" class="filter-option pull-left appformer-live-search-button-text">- Select Editor
```
[the remark this answers is not in the document]
Comment (RishiRajAnand) [reply uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/dropdown/LiveSearchDropDownView.html]: Yes, It is. Somehow, It went unnoticed by me. Thanks @paulovmr  for this observation.

**Comment by pefernan** [comment]
jenkins execute full downstream build please

**Comment by ederign** [comment]
Fails are not related.

**Comment by pefernan** [comment]
@ederign  I had no time to test... :S

**Pull request merged**